resources/js/components/ui/chart.tsx

```
@@ -579,9 +579,9 @@ const ChartLegendContent = ({
             key={key}
             id={key}
             className={twMerge(
-              "*:data-[slot=icon]:-mx-0.5 flex items-center gap-2 rounded-sm px-2 py-1 text-muted-fg *:data-[slot=icon]:size-2.5 *:data-[slot=icon]:shrink-0 *:data-[slot=icon]:text-muted-fg",
-              "selected:bg-secondary/70 selected:text-secondary-fg",
-              "hover:bg-secondary/70 hover:text-secondary-fg",
+              "*:data-[slot=icon]:-mx-0.5 flex items-center gap-2 rounded-sm px-2 py-1 text-muted-foreground *:data-[slot=icon]:size-2.5 *:data-[slot=icon]:shrink-0 *:data-[slot=icon]:text-muted-foreground",
```

Biome (2.4.13)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)